### PR TITLE
Jn photogram industrial 2

### PR DESCRIPTION
**Pull request by jniewiel**

### CODE REVIEW

**Review by aborov — APPROVED** [review]
Looks good!

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+#  index_comments_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (author_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: "User"
```
Comment (thierrychau) [inline app/models/comment.rb]: Can you add a counter_cache?

File: app/models/comment.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  belongs_to :photo
```
Comment (thierrychau) [inline app/models/comment.rb]: Can you add a counter_cache?

File: app/models/like.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+#  index_likes_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
```
Comment (thierrychau) [inline app/models/like.rb:23]: Can you add a validation to make sure a user is not able to like the same photo more than once?

**Review by thierrychau — CHANGES_REQUESTED** [review]
Please see below comments

**Review by thierrychau — APPROVED** [review]
Looks great, good to merge.

**Pull request merged**